### ISSUE TITLE
Make list-item component accessible for keyboard navigation

### ISSUE THREAD

**Issue by aniri** [code4romania/taskforce-fe-components#168]
It should be possible to navigate between list items using the tab key.

Suggestions:
- add `tabindex` property - https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets#Using_tabindex
- user aria attributes - https://www.w3.org/TR/wai-aria-practices/

**Comment by danmana** [comment]
I can take this
